From d924fe1e0d428cf56f301cccf3f0e7848a5a93c8 Mon Sep 17 00:00:00 2001
From: Brad Geltz <brad.geltz@intel.com>
Date: Wed, 7 Oct 2020 12:17:19 -0700
Subject: [PATCH 2/6] Enable MPI in Intel build

#
#  Copyright (c) 2015 - 2023, Intel Corporation
#  SPDX-License-Identifier: BSD-3-Clause
#

- Use the build environment variables.

Signed-off-by: Brad Geltz <brad.geltz@intel.com>
---
 src/Makefile.intel.openmp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/Makefile.intel.openmp b/src/Makefile.intel.openmp
index 390f163..df085d9 100644
--- a/src/Makefile.intel.openmp
+++ b/src/Makefile.intel.openmp
@@ -19,13 +19,13 @@ MINIFE_MATRIX_TYPE = -DMINIFE_CSR_MATRIX
 CFLAGS = -O3 -fopenmp
 CXXFLAGS = $(CFLAGS)
 
-CPPFLAGS = -I. -I../utils -I../fem $(MINIFE_TYPES) $(MINIFE_MATRIX_TYPE)
+CPPFLAGS = -I. -I../utils -I../fem $(MINIFE_TYPES) $(MINIFE_MATRIX_TYPE) -DHAVE_MPI
 
 LDFLAGS=
 LIBS=
 
-CXX=icpc
-CC=icc
+CXX=$(MPICXX)
+CC=$(MPICC)
 
 #-----------------------------------------------------------------------
 
-- 
2.26.2

